src/fatxfs_fuse.c

```diff
@@ -617,19 +610,24 @@ void fatx_fuse_print_usage(void)
     /* Print basic usage */
     fprintf(stderr, "FATXFS - Userspace FATX Filesystem Driver\n\n");
     fprintf(stderr, "Usage: %s <device> <mountpoint> [<options>]\n", prog_short_name);
-    fprintf(stderr, "   or: %s <device> <mountpoint> --drive=c|e|x|y|z [<options>]\n", prog_short_name);
+    fprintf(stderr, "   or: %s <device> <mountpoint> --drive=c|e|x|y|z|f [<options>]\n", prog_short_name);
     fprintf(stderr, "   or: %s <device> <mountpoint> --offset=<offset> --size=<size> [<options>]\n\n", prog_short_name);
-    fprintf(stderr, "General Options:\n"
-                    "    -o opt, [opt...]       mount options\n"
-                    "    -h --help              print help\n"
-                    "    -V --version           print version\n\n"
+    fprintf(stderr, "General options:\n"
+                    "    -o opt, [opt...]               mount options\n"
+                    "    -h --help                      print help\n"
+                    "    -V --version                   print version\n\n"
                     "FATXFS options:\n"
-                    "    --drive=<letter>       mount a partition by its drive letter\n"
-                    "    --offset=<offset>      specify the offset (in bytes) of a partition manually\n"
-                    "    --size=<size>          specify the size (in bytes) of a partition manually\n"
-                    "    --sector-size=<size>   specify the size (in bytes) of a device sector (default is 512)\n"
-                    "    --log=<log path>       enable fatxfs logging\n"
-                    "    --loglevel=<level>     control the log output level (a higher value yields more output)\n\n");
+                    "    --drive=<letter>               mount a partition by its drive letter\n"
+                    "    --offset=<offset>              specify the offset (in bytes) of a partition manually\n"
+                    "    --size=<size>                  specify the size (in bytes) of a partition manually\n"
+                    "    --sector-size=<size>           specify the size (in bytes) of a device sector (default is 512)\n"
+                    "    --log=<log path>               enable fatxfs logging\n"
+                    "    --loglevel=<level>             control the log output level (a higher value yields more output)\n\n"
+                    "Disk formatting options:\n"
+                    "    --format=<format>              specify the format (retail, f-takes-all) to initialize the device to\n"
```

Comment:
It would be nice to support formatting XMU memory card partitions, which take the whole disk, use a sector size of 4096, and sectors per cluster of 4. But we can defer it to a follow up